projects/plugins/jetpack/modules/comments/subscription-modal-on-comment/class-jetpack-subscription-modal-on-comment.php

```diff
@@ -164,21 +161,9 @@ class="jetpack-subscription-modal__iframe"
 		<p class='has-text-align-center' style='margin-top:4px;margin-bottom:0px;font-size:15px'>$subscribe_text</p>
 		<!-- /wp:paragraph -->
 
-		<!-- wp:group {"style":{"spacing":{"top":"32px","bottom":"32px","left":"32px","right":"32px"},"margin":{"top":"0","bottom":"0"}},"border":{"color":"#dddddd","width":"1px"}},"layout":{"type":"constrained","contentSize":"450px"}} -->
-		<form class="jetpack-subscription-modal__form">
-			<input
-				class="jetpack-subscription-modal__form-email"
-				required="required"
-				type="email"
-				name="email"
-				style="border-width: 1px !important;"
-				/>
-			<button
-				class="jetpack-subscription-modal__form-submit"
-				type="submit"
-				name="jetpack_subscriptions_widget" >$subscribe_button</button>
-		</form>
-		<!-- /wp:group -->
+		<!-- wp:jetpack/subscriptions {"borderRadius":50,"className":"is-style-compact","appSource":"atomic-subscription-modal-lo"} /-->
+
+		<p class="has-text-align-center jetpack-subscription-modal__close" style="margin-top:20px;font-size:14px"><a href="#">$continue_reading</a></p>
```

Comment:
Would be good to have the full block markup here for `do_blocks` purposes as well as for future if we allow customer to edit the contents of this modal in block editor, it'll be easier:

```html
<!-- wp:paragraph {"align":"center","style":{"spacing":{"margin":{"top":"20px"}},"typography":{"fontSize":"14px"}},"className":"jetpack-subscription-modal__close"} -->
<p class="has-text-align-center jetpack-subscription-modal__close" style="margin-top:20px;font-size:14px"><a href="#">$continue_reading</a></p>
<!-- /wp:paragraph -->
```

Reply:
That's great, I updated the PR with your suggestion. TIL